src/app/app.component.html

```diff
@@ -102,12 +96,10 @@ <h2>Contact Us</h2>
       <div class="flex gap-x-4">
         <abds-input name="street" pattern="\w+(\s\w+){2,}" required>Street</abds-input>
         <abds-input name="city" pattern="[A-Z|a-z|\s]+" required>City</abds-input>
-        <abds-select id="states" label="State" name="state" placeholder="Select a state" required>
-          <!--  Default options if  -->
-          <abds-select-option value="cali">California</abds-select-option>
-          <abds-select-option value="denver">Denver</abds-select-option>
-          <abds-select-option value="new york">New York</abds-select-option>
-          <abds-select-option value="tx">Texas</abds-select-option>
+        <abds-select *ngIf="states" id="states" label="State" name="state" placeholder="-- Select a state --" required>
+          <abds-select-option *ngFor="let state of states" value="{{ state.state_name.toLowerCase() }}">
```

Comment:
good use of angular operators. This looks a lot like Vue to me so that's interesting.